spec/unit/lockfile/parser_spec.rb

```diff
@@ -57,8 +57,7 @@ module Librarian
       it "should give a dependency with the expected requirement" do
         dependency = resolution.dependencies.first
 
-        # Note: it must be this order because this order is lexicographically sorted.
-        expect(dependency.requirement.to_s).to eq "~> 1.1, != 1.2.6"
+        expect(dependency.requirement).to eq Librarian::Dependency::Requirement.new(["~> 1.1", "!= 1.2.6"])
```

Comment:
This keeps bugging me. I'll need to look at the equivalence implementation on the requirement.

Reply:
Looks like it no longer shows on Github Actions. Perhaps it had to do with Xenial on Travis.